dist/rpm/openQA.spec

```diff
@@ -257,13 +257,24 @@ Covering both openQA and also os-autoinst test engine.
 %package auto-update
 Summary:        Automatically upgrade and reboot the system when required
 Group:          Development/Tools/Other
+Requires:       %{name}-common
 Requires:       curl
 Requires:       rebootmgr
 
 %description auto-update
 Use this package to install and enable a systemd service for nightly upgrading
 and rebooting the system if devel:openQA packages are stable.
 
+%package continuous-update
+Summary:        Continuously update packages from devel:openQA
+Group:          Development/Tools/Other
+Requires:       %{name}-common
+Requires:       curl
+
+%description continuous-update
+Use this package to install and enable a systemd service for continuously upgrading
```

Comment:
We should describe here the relation to "auto-update".

As auto-update does a `zypper dup` without repo restrictions and then even triggers a full system reboot they are complementary, not alternatives.

Please also extend the documentation. See
https://github.com/os-autoinst/openQA/blob/master/docs/Installing.asciidoc#automatic-system-upgrades-and-reboots-of-openqa-hosts= about the according auto-update section where you could extend.

Reply:
> they are complementary, not alternatives

Yes. I also have both mechanisms still in place on openqaworker7 and also don't see it as part of the initial ticket to remove the existing auto update.